Framework/PythonInterface/mantid/plots/datafunctions.py

```diff
@@ -549,7 +549,7 @@ def _workspace_indices(y_bins, workspace):
 
 def _workspace_indices_maxpooling(y_bins, workspace):
     from mantid.simpleapi import Integration
-    summed_spectra_workspace = Integration(workspace)
+    summed_spectra_workspace = Integration(workspace, OutputWorkspace="__summed_spectra_workspace")
```

Comment:
Why have we changed the name for this - if users don't want to see it in the ADS would it be better to pass it around by handle instead

`summed_spectra_workspace = Integration(workspace, StoreInADS=False)`

Reply:
Ah yes, forgot there was a `StoreInAds` flag 🤦‍♂️